### PR TITLE
Reinstate sorting and filtering options - closes #3184

### LINKED ISSUE #3184: Reinstate date values, sorting and filtering options
### Description
Changed made by #3182 were temporary and we should revert them after changes defined by #2890 are implemented.
Refer to #3185 for details of the changes.

### Acceptance Criteria
- Display date from tx list
- Display date from tx details
- Display sorting by date
- Display filtering by date
- Reinstate related e2e tests

### PR DESCRIPTION
**Pull request by reyraa**
### What was the problem?
This PR resolves #3184

### How was it solved?
- Remove outgoing/incoming tabs
- Fixed filters data structure
- Fix filter conversion issue in getTransactions method

### CODE REVIEW

File: src/components/screens/wallet/index.js [hunk 1]
```diff
@@ -100,6 +104,7 @@ const Wallet = ({ t, history }) => {
           tabName={t('Transactions')}
           tabId="Transactions"
           t={t}
+          isWallet={history.location.pathname === '/wallet'}
```
Comment (isalga) [inline src/components/screens/wallet/index.js:107]: Prefer to use `routes.wallet.path` from `constants/routes.js`

File: src/components/screens/wallet/transactions/transactions.css [hunk 2]
```diff
@@ -1,5 +1,17 @@
 @import '../../../../app/mixins.css';
 
+.wrapper {
+  & > header {
+    justify-content: flex-end !important;
+    padding: 0 !important;
+    height: 20px !important;
```
Comment (isalga) [inline src/components/screens/wallet/transactions/transactions.css:7]: I would prefer to avoid using `!important` if possible.

**Review by isalga — CHANGES_REQUESTED** [review]

File: src/components/screens/wallet/transactions/transactions.css (hunk 2 above)
Comment (reyraa) [reply src/components/screens/wallet/transactions/transactions.css:7]: The rule that overrides this has so many levels, it's unreasonably hard to override it in this file.

File: src/components/screens/wallet/index.js (hunk 1 above)
Comment (reyraa) [reply src/components/screens/wallet/index.js]: Done

**Review by isalga — APPROVED** [review]

**Pull request merged**